src/App.js

```diff
@@ -1,36 +1,53 @@
-import React, { useState, useEffect } from 'react';
-import AddTodoForm from './AddTodoForm';
-import TodoList from './TodoList';
-
-const useSemiPersistentState = () => {
-  const [todoList, setTodoList] = useState(JSON.parse(localStorage.getItem("savedTodoList")) || []);
-
-  useEffect(() => {
-    localStorage.setItem("savedTodoList", JSON.stringify(todoList))
-  }, [todoList] )
-
-  return [todoList, setTodoList]
-}
+import React, { useState, useEffect } from "react";
+import AddTodoForm from "./AddTodoForm";
+import TodoList from "./TodoList";
 
 function App() {
-  const [todoList, setTodoList] = useSemiPersistentState()
-
-  const addTodo = (newTodo) => {
-    setTodoList([...todoList, newTodo]);
-  }
-
-  const removeTodo = (id) => {
-    const updatedTodos = todoList.filter(todo => todo.id !== id);
-    setTodoList(updatedTodos);
-  }
-
-  return (
-   <> 
-     <h1>Todo List</h1>
-     <AddTodoForm onAddTodo={addTodo} />
-     <TodoList todoList={todoList} onRemoveTodo={removeTodo}/>
-   </>
-  );
+	const [todoList, setTodoList] = useState([]);
+	const [isLoading, setIsLoading] = useState(true);
+
+	useEffect(() => {
+		const url = `https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/Default`;
+		try {
+			fetch(url, {
+				headers: {
+					Authorization: `Bearer ${process.env.REACT_APP_AIRTABLE_API_KEY}`,
+				},
+			})
+				.then((result) => result.json())
+				.then((result) => setTodoList(result.records));
+			setIsLoading(false);
+		} catch (error) {
+			console.log(error.message);
+		}
+	}, []);
```

Comment:
Cool! Good use of try/catch. In a production environment, I often use [toast messages](https://uxplanet.org/toast-notification-or-dialog-box-ae32ad53106d) to let the user know (in user-friendly language of course!) that something went awry.

Reply:
Oh thanks! I'm excited to try that out.